templates/cluster-template-aks-multi-tenancy.yaml

```diff
@@ -23,6 +23,9 @@ metadata:
   name: ${CLUSTER_NAME}
   namespace: default
 spec:
+  addonProfiles:
```

Comment:
Ideally we shouldn't enable features for tests directly in the flavors as those are also exposed as examples for users, but we can add patches in the test/ directory for that sort of stuff

since this is likely to merge before #2143 and I'm removing this template anyways, let's keep this for now and I'll do some cleanup in my PR.